.github/ISSUE_TEMPLATE/avm_module_issue.yml

```diff
@@ -140,6 +140,7 @@ body:
         - "avm/res/key-vault/vault"
         - "avm/res/kubernetes-configuration/extension"
         - "avm/res/kubernetes-configuration/flux-configuration"
+        - "avm/res/kubernetes/connected-clusters"
```

Comment:
I think there has been a mistake in onboarding this module with a plural form. As per spec [RMNFR1 ](https://azure.github.io/Azure-Verified-Modules/spec/RMNFR1/) module names should be in a singular form, hence this module should be named `avm/res/kubernetes/connected-cluster` also consistently with other clusters (e.g. `avm/res/container-service/managed-cluster` or `avm/res/kusto/cluster`)

@matebarabas @xhy8759 to comply with this spec, we should:
- onboard the correct singular form to MAR (core team)
- update the AVM csv (core team)
- update the Azure team names
- update the code and folder names in this PR accordingly

Reply:
@eriqua, same as discussed here https://github.com/Azure/Azure-Verified-Modules/issues/1791#issuecomment-2667384000 - I'm fixing this typo in the index with the next update; I'll also get the MAR record patched. Apologies for my confusion; I'll let you know when this is fixed. Thanks again for catching this!